src/VirtoCommerce.ElasticAppSearch.Core/Models/Api/Search/SearchResultMetadata.cs

```diff
@@ -0,0 +1,18 @@
+using VirtoCommerce.ElasticAppSearch.Core.Models.Api.Engines;
+
+namespace VirtoCommerce.ElasticAppSearch.Core.Models.Api.Search;
+
+public record SearchResultMetadata
+{
+    public string[] Alerts { get; set; }

```

Comment:
Maybe ICollection<string> would be better?

Reply:
I'm not sure what Newtonsoft.Json can deserialize to ICollection, but even if yes, it has no meaning: API objects `*Result*` is only returned by API and should be immutable